app/Domain/Tickets/Templates/showAll.blade.php

```diff
@@ -126,261 +126,238 @@
+
+                                    @foreach ($tpl->get('milestones') as $milestone)
+                                        <x-global::actions.dropdown.item href="javascript:void(0);" :data-label="$tpl->escape($milestone->headline)"
+                                            :data-value="$row['id'] .
+                                                '_' .
+                                                $milestone->id .
+                                                '_' .
+                                                $tpl->escape($milestone->tags)" :id="'ticketMilestoneChange' . $row['id'] . $milestone->id" :style="'background-color:' . $tpl->escape($milestone->tags)">
+                                            {{ $tpl->escape($milestone->headline) }}
                                         </x-global::actions.dropdown.item>
                                     @endforeach
-
-                                </x-global::content.context-menu>
-
+                                </x-global::forms.dropdownPill>
                             </td>
+
                             <td
                                 data-order="<?= $row['storypoints'] ? $efforts['' . $row['storypoints'] . ''] ?? '?' : $tpl->__('label.story_points_unkown') ?>">
-                                @php
-                                    // Determine the label text based on conditions
-                                    $labelText =
-                                        $row['storypoints'] != '' && $row['storypoints'] > 0
-                                            ? $efforts[$row['storypoints']] ?? $row['storypoints']
-                                            : __('label.story_points_unkown');
-                                @endphp
-
-                                <!-- Use the determined label text in the dropdown component -->
-                                <x-global::content.context-menu :label-text="htmlspecialchars($labelText) .
-                                    '&nbsp;<i class=\'fa fa-caret-down\' aria-hidden=\'true\'></i>'" contentRole="link" position="bottom"
-                                    align="start" class="label-default effort f-left">
-
-                                    <!-- Menu Header -->
-                                    <li class="nav-header border">{{ __('dropdown.how_big_todo') }}</li>
-
-                                    <!-- Dynamic Effort Options -->
-                                    @foreach ($efforts as $effortKey => $effortValue)
-                                        <x-global::actions.dropdown.item href="javascript:void(0);"
-                                            data-value="{{ $row['id'] . '_' . $effortKey }}"
-                                            id="ticketEffortChange{{ $row['id'] . $effortKey }}">
-                                            {{ $effortValue }}
+                                <div class="dropdown ticketDropdown effortDropdown show">
```

Comment:
Why this line here? That is part of the dropdownPill component

Reply:
removed this, thought this was necessary